src/handlers/cell.ts

```diff
@@ -21,31 +21,42 @@ const LINE_HIGHLIGHT_CLASS = 'jp-breakpoint-line-highlight';
 
 export class CellManager implements IDisposable {
   constructor(options: CellManager.IOptions) {
-    this._debuggerModel = options.debuggerModel;
     this._debuggerService = options.debuggerService;
-    this.breakpointsModel = options.breakpointsModel;
+    this.onModelChanged();
+    this._debuggerService.modelChanged.connect(() => this.onModelChanged());
```

Comment:
We could also write it like to keep the usual style: 

```suggestion
    this._debuggerService.modelChanged.connect(this.onModelChanged, this);
```